### ISSUE TITLE
Define the data model for the camps

### ISSUE THREAD

**Issue by bobinson** [raksha-life/Project-Management#1]
- We need to define a data model for the release camps using HDX as a referee

- This will ensure that the camp module has interoperability with other modules

- https://github.com/HXLStandard/libhxl-python

- https://hxlstandard.org/

- the data exchange itself should be happening using OpenAPI 2.0 between apps

- HDX and HXL will be used across all the modules

**Comment by cibingeorge** [comment]
no. Just do standard db models.. what are the tables in a mysql db(im not proposing mysql here ..any traditional db) ..what are fields, what column types lengths and constraints, and what are the relationships.

**Comment by bobinson** [comment]
there is no app.
